tests/support/tools_test.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2025 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import torch
+from iree.turbine.support.tools import iree_tool_prepare_input_args
+import tempfile
+from pathlib import Path
+
+
+def test_iree_tool_prepare_input_args():
+    arg0 = torch.tensor([1.1, 2.2, 3.3, 4.4], dtype=torch.bfloat16)
+    arg1 = torch.tensor([[4, 5], [6, 7]], dtype=torch.int8)
+    args = [arg0, arg1]
+    with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
Could use the `tmp_path` fixture here instead: https://docs.pytest.org/en/stable/how-to/tmp_path.html

Reply:
Done.
I did not know about that built-in fixture.